opencog/util/Logger.cc

```diff
@@ -273,7 +274,8 @@ Logger::Logger(const std::string &fname, Logger::Level level, bool tsEnabled)
 }
 
 Logger::Logger(const Logger& log)
-    : error(*this), warn(*this), info(*this), debug(*this), fine(*this)
+    : error(*this), warn(*this), info(*this), debug(*this), fine(*this),
+      threadIdEnabled(false)
```

Comment:
Shouldn't `threadIdEnabled` just be assigned the value of `log.threadIdEnabled` instead of being initialized to false?